polar-core/src/vm.rs

```diff
@@ -509,6 +545,7 @@ impl PolarVirtualMachine {
             Goal::TraceRule { trace } => {
                 if let Node::Rule(rule) = &trace.node {
                     self.log_with(
+                        LogLevel::Trace,
```

Comment:
Another "downgrade" from Info -> Trace. Highlighting all of these to make sure they're intentional shifts